src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -175,16 +236,20 @@
   {%- endif -%}
 
   {%- if _input.input_type == 'range' -%}
-    {% include '@ecl/range/range.html.twig' with _input only %}
+    {% include '@ecl/range/range.html.twig' with _input|merge({
+      extra_attributes: _input_extra_attributes,
+    }) only %}
   {%- endif -%}
 
   {%- if _input.input_type == 'rating-field' -%}
     {% include '@ecl/rating-field/rating-field.html.twig' with _input only %}
   {%- endif -%}
 
   {%- if _input.input_type == 'file' -%}
-    {% include '@ecl/file-upload/file-upload.html.twig' with _input only %}
+    {% include '@ecl/file-upload/file-upload.html.twig' with _input|merge({
+      extra_attributes: _input_extra_attributes,
```

Comment:
since we do this for all the templates we include, wouldn't it be easier to do it at once instead of adding them in each instance of the includes we have?